transport/src/main/java/io/camunda/zeebe/transport/stream/impl/RemoteStreamPusher.java

```diff
@@ -52,11 +52,15 @@ final class RemoteStreamPusher<P extends BufferWriter> {
 
   public void pushAsync(
       final P payload, final RemoteStreamErrorHandler<P> errorHandler, final StreamId streamId) {
-    Objects.requireNonNull(payload, "must specify a payload");
     Objects.requireNonNull(errorHandler, "must specify a error handler");
```

Comment:
:thought_balloon: I've left it out, because, well, if you have no error handler there's literally no way to report it, so I think that should break everything :smile:

Reply:
Makes sense thanks